### ISSUE TITLE
validation criteria

### ISSUE THREAD

**Issue by bmcfee** [bmcfee/augment_music#3]
Toward #2 : we need some model selection criterion, ie, some function to minimize over the validation set to select among models.

There are a few options here:
1. model objective (ie, mean binary entropy)
2. actual classifier performance, such as
   
   a. [f1_score](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.f1_score.html#sklearn.metrics.f1_score) (macro, micro, or samples?)
   
   b. [hamming loss](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.hamming_loss.html#sklearn.metrics.hamming_loss)
   
   c. [accuracy score](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.accuracy_score.html#sklearn.metrics.accuracy_score)
   
   d. [lrap](http://scikit-learn.org/stable/modules/generated/sklearn.metrics.label_ranking_average_precision_score.html#sklearn.metrics.label_ranking_average_precision_score)
